test/integration/error.test.js

```diff
@@ -34,6 +38,9 @@ describe(Support.getTestDialectTeaser('Sequelize Errors'), function () {
       expect(validationError).to.be.instanceOf(Sequelize.ValidationError);
       expect(validationError).to.be.instanceOf(Error);
       expect(validationError).to.have.property('name', 'SequelizeValidationError');
+      console.log(validationError.message);
+      console.log(typeof(validationError.message));
+      expect(validationError.message).to.be('notNull Violation: <field name> cannot be null,\nstring violation: <field name> cannot be an array or an object');
```

Comment:
> I think `to.match` is what you are looking for.
Thanks. I'll try it.

> Also, please squashs your commits into one
OK, but are you talking about two "WIP" commits? Or should I squash all commits into one commit?